### PR TITLE
change strstr/substr with strpos when possible

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: src/Helper/AbstractHtmlElement.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ protected function htmlAttribs($attribs)
         foreach ((array) $attribs as $key => $val) {
             $key = $escaper($key);
 
-            if (('on' == substr($key, 0, 2)) || ('constraints' == $key)) {
+            if ((0 === strpos($key, 'on')) || ('constraints' == $key)) {
```
Comment (webimpress) [inline src/Helper/AbstractHtmlElement.php]: We can remove useless brackets here as well

Comment (samsonasik) [reply src/Helper/AbstractHtmlElement.php]: done ;)

**Comment by weierophinney** [comment]
Thanks, @samsonasik

**Comment by weierophinney** [comment]
Merged to develop for release with 2.11.0

**Pull request closed without merging**